sshmuxer/requests.go

```diff
@@ -171,6 +177,10 @@ func handleRemoteForward(newRequest *ssh.Request, sshConn *utils.SSHConnection,
 			return
 		}
 
+		//Reply that the session is successfully set up with the port that was assigned (which may be different than what was requested)
+		channelForwardReplyPayload := ssh.Marshal(channelForwardReply{bindPort})
+		newRequest.Reply(true, channelForwardReplyPayload)
```

Comment:
According to the rfc, it looks like the port reply is only sent as part of a `0` port allocation request (random port). Are there any side effects of always returning this in your testing?

Reply:
Tested with Openssh, putty, and a golang-based ssh client, didn't have an issue.

I could add more check but as I recall it was difficult trying to find out if the server assigned a random port or not because the info wasn't in the scope, I'll look into it.

Reply:
My best guess is it wouldn't cause an issue. Just was hoping for an anecdotal "looks good!" to make sure my suspicions were confirmed!